zhaquirks/tuya/TS011F_plug.py

```diff
@@ -0,0 +1,112 @@
+"""TS011F plug."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy, Groups, Identify, OnOff, Ota, Scenes, Time
+from zigpy.zcl.clusters.homeautomation import ElectricalMeasurement
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODEL,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+class TuyaZBMetering(CustomCluster, Metering):
+    """TuyaZBMetering cluster implementation. This implementation divides the KwH for tuya devices"""
+
+    cluster_id = Metering.cluster_id
+    current_summ_delivered = 0x0000
+
+    def _update_attribute(self, attrid, value):
+        if attrid == self.current_summ_delivered:
+            value = value / 100
+        super()._update_attribute(attrid, value)
```

Comment:
Check https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/edpwithus/__init__.py Can you test with 
```suggestion
    MULTIPLIER = 0x0301
    DIVISOR = 0x0302
    _CONSTANT_ATTRIBUTES = {MULTIPLIER: 1, DIVISOR: 100}
```

Reply:
Hi, not working anymore. Value isn't divided by 100 anymore